inc/switchtec/mrpc.h

```diff
@@ -90,6 +90,7 @@ enum mrpc_cmd {
 	MRPC_SN_VER_GET = 265,
 	MRPC_DBG_UNLOCK = 266,
 	MRPC_BOOTUP_RESUME = 267,
+	MRPC_SECURITY_INFO_GET = 268,
```

Comment:
If it's a Gen5 version of MRPC_SECURITY_CONFIG_GET, maybe we can use suffix _GEN5 to name it. (and also align with other naming in this patch)